hack/extra_rpm_repositories.yml

```diff
@@ -37,3 +37,5 @@ extras:
   -  "appstream"
   -  "appstream-sources"
   -  "codeready-builder"
+  -  "rhel-10-flatpak-app-build"
```

Comment:
@ralphbean, @rhartman93 these look like they should be coming from https://access.redhat.com/security/data/meta/v1/repository-to-cpe.json ?